bot/cogs/filtering.py

```diff
@@ -125,6 +99,23 @@ def __init__(self, bot: Bot):
 
         self.bot.loop.create_task(self.reschedule_offensive_msg_deletion())
 
+    def _get_allowlist_items(self, list_type: str, *, allowed: bool, compiled: Optional[bool] = False) -> list:
+        """Fetch items from the allow_deny_list_cache."""
+        items = self.bot.allow_deny_list_cache.get(f"{list_type.upper()}.{allowed}", [])
+
+        if compiled:
+            return [re.compile(fr'{item.get("content")}', flags=re.IGNORECASE) for item in items]
```

Comment:
Would've been a bit nicer to cache the compiled regular expressions like they were before, but I think doing that now would get kinda ugly. I don't think compiling them serves much benefit if you discard them after every use.

Reply:
yeah, I should probably refactor this to just use non-compiled expressions. will do.

Reply:
Fixed in 4e1609695762524bc707b6a8d39e88c2710cff6b